tools/ci/ci_manifest.sh

```diff
@@ -0,0 +1,9 @@
+set -ex
+
+SCRIPT_DIR=$(dirname $(readlink -f "$0"))
+WPT_ROOT=$(readlink -f $SCRIPT_DIR/../..)
+cd $WPT_ROOT
+
+mkdir -p ~/meta
```

Comment:
I don't understand the use of ~ here, where will the file actually end up? Is it in the wpt repo itself?

Reply:
It is generated in the `$HOME/meta` directory on the travis machine, and then finally gets uploaded to the "releases" part of the GitHub repo (if it works correctly :). But since it's not tagged  I don't think it will show up as an actual release. But I hope we can still download it.

Reply:
OK, as long as it's not in the wpt repo itself, I have no concerns. Let's see what happens :)